packages/slice-machine/src/legacy/components/Forms/CreateCustomTypeModal/CreateCustomTypeModal.tsx

```diff
@@ -165,10 +165,18 @@ export const CreateCustomTypeModal: React.FC<CreateCustomTypeModalProps> = ({
           })} name is already taken.`;
         }
 
+        if (["update", "insert"].includes(label.toLowerCase())) {
```

Comment:
⚠️ Isn't this a bit overly strict? Do we want to prevent customtypes/slices containing these? What would be the issue of having a label with the words `Inserted`, `Insertable`, `Insertion`, `Updated`, `Updater`, `Self-update`, `Updateable`, etc?

💡 Some alternatives:
1. The least strict: Maybe we should only block if it's just `insert` or `update`, and not a word containing it. 
2. Only block if the word is by itself, even on a sentence (e.g. `insert words` or `word update`), by using a regex like `/\b(insert|update)\b/gi`.

Reply:
@jomifepe I'm not sure if I understand the issue, here we check only if we match whole word  to be either 'update' or 'insert' not a part of it, so we still allow all of examples you provided

Reply:
Nevermind, I misread the code 😜